src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  curentKey: string,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    curentKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ curentKey: event.key });
+    });
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ curentKey: event.key });
+    });
+  }
+
+  render() {
+    const { curentKey } = this.state;
+
+    return (
+      <div className="App">
+        {(curentKey === ''
```

Comment:
This condition would be enough,  use  `!` here.
```suggestion
        {(!curentKey
```